src/components/Editor/Tabs.js

```diff
@@ -12,7 +12,9 @@ import { getFilename, isPretty } from "../../utils/source";
 import classnames from "classnames";
 import actions from "../../actions";
 import CloseButton from "../shared/Button/Close";
-const PaneToggleButton = createFactory(require("../shared/Button/PaneToggle"));
+const PaneToggleButton = createFactory(
```

Comment:
We can do this, and I have another possibility in mind, 
move createFactory inside component itself, so that way we need not to call createFactory every time we import a component.
Also @amitzur agrees that this way we can easily transition to JSX.